.github/workflows/ci.yml

```diff
@@ -0,0 +1,17 @@
+name: test
+on: [push]
```

Comment:
This means the workflow will trigger when a branch in this repository (not in a fork) is pushed to. So when an external contributor opens a new pull request, it won't trigger the workflow.

Reply:
Changed to follow the factory-boy example.